<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Tooling to summarize packages in review at Bioconductor • BiocRevSumm</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="bootstrap-toc.css">
<script src="bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="pkgdown.css" rel="stylesheet">
<script src="pkgdown.js"></script><meta property="og:title" content="Tooling to summarize packages in review at Bioconductor">
<meta property="og:description" content="Use LLM to summarize packages in review at Bioconductor">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">


    <div class="container template-home">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="index.html">BiocRevSumm</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.0.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="articles/BiocRevSumm.html">Get started</a>
</li>
<li>
  <a href="reference/index.html">Reference</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/vjcitn/BiocRevSumm/" class="external-link">
    <span class="fab fa-github fa-lg"></span>

  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->



      </header><div class="row">
  <div class="contents col-md-9">
<div class="section level1">
<div class="page-header"><h1 id="biocrevsumm">BiocRevSumm<a class="anchor" aria-label="anchor" href="#biocrevsumm"></a>
</h1></div>
<p>The intent of this package is to analyze metadata about the series of repositories <a href="https://github.com/bioconductor/contributions/issues" class="external-link">in review at Bioconductor</a>. We include code to interrogate the contribution issues, summarize package capabilities, and categorize all packages in review. Currently OpenAI LLMs are used to create summaries and classifications of packages.</p>
<p>The listing below is a demonstration created with the package on Nov 19 2025. There are clear deficiencies of labeling, and improved prompting and tooling are needed. Some repositories are excluded because insufficient metadata is present in the DESCRIPTION or README.md.</p>
<div class="section level2"><h2 id="example-grouping-of-github-repositories-under-review-at-bioconductor">Example grouping of GitHub Repositories under review at Bioconductor<a class="anchor" aria-label="anchor" href="#example-grouping-of-github-repositories-under-review-at-bioconductor"></a>
</h2></div>
<div class="section level2">
<h2 id="1-single-cell-rna-seq-analysis">1. Single-Cell RNA-Seq Analysis<a class="anchor" aria-label="anchor" href="#1-single-cell-rna-seq-analysis"></a>
</h2>
<ul>
<li>
<strong>posDemux</strong>: Demultiplexing and filtering sequence reads with combinatorial barcodes.<ul><li>URL: <a href="https://github.com/yaccos/posDemux" class="external-link uri">https://github.com/yaccos/posDemux</a>
</li></ul>
</li>
<li>
<strong>fourSynergy</strong>: Ensemble algorithm for analyzing 4C-seq data.<ul><li>URL: <a href="https://github.com/sophiewind/fourSynergy" class="external-link uri">https://github.com/sophiewind/fourSynergy</a>
</li></ul>
</li>
<li>
<strong>CellMentor</strong>: Dimensional reduction using supervised cell type-aware non-negative matrix factorization.<ul><li>URL: <a href="https://github.com/petrenkokate/CellMentor/" class="external-link uri">https://github.com/petrenkokate/CellMentor/</a>
</li></ul>
</li>
<li>
<strong>hammers</strong>: Tools for single-cell RNA sequencing (scRNA-seq) data analysis.<ul><li>URL: <a href="https://github.com/andrei-stoica26/hammers" class="external-link uri">https://github.com/andrei-stoica26/hammers</a>
</li></ul>
</li>
</ul>
</div>
<div class="section level2">
<h2 id="2-tumor-and-mutational-data-simulation">2. Tumor and Mutational Data Simulation<a class="anchor" aria-label="anchor" href="#2-tumor-and-mutational-data-simulation"></a>
</h2>
<ul>
<li>
<strong>ClonalSim</strong>: Simulating the clonal evolution of tumors with realistic sequencing noise.<ul><li>URL: <a href="https://github.com/gbucci/ClonalSim" class="external-link uri">https://github.com/gbucci/ClonalSim</a>
</li></ul>
</li>
<li>
<strong>MutSeqRData</strong>: Experimental data package for mutational sequencing analysis.<ul><li>URL: <a href="https://github.com/EHSRB-BSRSE-Bioinformatics/MutSeqRData" class="external-link uri">https://github.com/EHSRB-BSRSE-Bioinformatics/MutSeqRData</a>
</li></ul>
</li>
</ul>
</div>
<div class="section level2">
<h2 id="3-metabolomics">3. Metabolomics<a class="anchor" aria-label="anchor" href="#3-metabolomics"></a>
</h2>
<ul>
<li>
<strong>MetaboAnnotatoR</strong>: Metabolite annotation of features from Liquid Chromatography-Mass Spectrometry datasets.<ul><li>URL: <a href="https://github.com/gggraca/MetaboAnnotatoR" class="external-link uri">https://github.com/gggraca/MetaboAnnotatoR</a>
</li></ul>
</li>
<li>
<strong>MetaProViz</strong>: Mechanistic hypotheses in metabolomics data by integrating literature.<ul><li>URL: <a href="https://github.com/saezlab/MetaProViz" class="external-link uri">https://github.com/saezlab/MetaProViz</a>
</li></ul>
</li>
<li>
<strong>MetabolomicsPipeline</strong>: Analyzing metabolomics data with subpathway analysis and other tools.<ul><li>URL: <a href="https://github.com/datalifecycle-ua/MetabolomicsPipeline/" class="external-link uri">https://github.com/datalifecycle-ua/MetabolomicsPipeline/</a>
</li></ul>
</li>
</ul>
</div>
<div class="section level2">
<h2 id="4-gene-and-protein-expression-analysis">4. Gene and Protein Expression Analysis<a class="anchor" aria-label="anchor" href="#4-gene-and-protein-expression-analysis"></a>
</h2>
<ul>
<li>
<strong>scrapple</strong>: Wrappers for single-cell RNA-Seq analysis.<ul><li>URL: <a href="https://github.com/libscran/scrapple" class="external-link uri">https://github.com/libscran/scrapple</a>
</li></ul>
</li>
<li>
<strong>TraianProt</strong>: Downstream analysis of quantitative proteomic data.<ul><li>URL: <a href="https://github.com/SamueldelaCamaraFuentes/TraianProt" class="external-link uri">https://github.com/SamueldelaCamaraFuentes/TraianProt</a>
</li></ul>
</li>
<li>
<strong>OAtools</strong>: Streamlined analysis of gene expression data on OpenArray platform.<ul><li>URL: <a href="https://github.com/uwvirology-ngs/OAtools" class="external-link uri">https://github.com/uwvirology-ngs/OAtools</a>
</li></ul>
</li>
</ul>
</div>
<div class="section level2">
<h2 id="5-chromatin-and-dna-interaction">5. Chromatin and DNA Interaction<a class="anchor" aria-label="anchor" href="#5-chromatin-and-dna-interaction"></a>
</h2>
<ul>
<li>
<strong>annoLinker</strong>: Annotation of genomic peaks using DNA interaction data.<ul><li>URL: <a href="https://github.com/jianhong/annoLinker" class="external-link uri">https://github.com/jianhong/annoLinker</a>
</li></ul>
</li>
<li>
<strong>SMTrackR</strong>: Visualization of protein-DNA binding states on sequenced DNA molecules.<ul><li>URL: <a href="https://github.com/satyanarayan-rao/SMTrackR" class="external-link uri">https://github.com/satyanarayan-rao/SMTrackR</a>
</li></ul>
</li>
</ul>
</div>
<div class="section level2">
<h2 id="6-genomic-visualization-and-enrichment-analysis">6. Genomic Visualization and Enrichment Analysis<a class="anchor" aria-label="anchor" href="#6-genomic-visualization-and-enrichment-analysis"></a>
</h2>
<ul>
<li>
<strong>GOfan</strong>: Visualization of Gene Ontology enrichment results using sunburst layout.<ul><li>URL: <a href="https://github.com/jianhong/GOfan" class="external-link uri">https://github.com/jianhong/GOfan</a>
</li></ul>
</li>
<li>
<strong>ClusterGVis</strong>: Clustering and visualization of gene expression data.<ul><li>URL: <a href="https://github.com/junjunlab/ClusterGVis" class="external-link uri">https://github.com/junjunlab/ClusterGVis</a>
</li></ul>
</li>
<li>
<strong>ImageArray</strong>: Handling large image arrays in digital pathology and microscopy.<ul><li>URL: <a href="https://github.com/BIMSBbioinfo/ImageArray" class="external-link uri">https://github.com/BIMSBbioinfo/ImageArray</a>
</li></ul>
</li>
</ul>
</div>
<div class="section level2">
<h2 id="7-data-processing-and-pipeline-tools">7. Data Processing and Pipeline Tools<a class="anchor" aria-label="anchor" href="#7-data-processing-and-pipeline-tools"></a>
</h2>
<ul>
<li>
<strong>tidyprint</strong>: Enhancing usability of ‘SummarizedExperiment’ objects within a tidy workflow.<ul><li>URL: <a href="https://github.com/tidyomics/tidyprint" class="external-link uri">https://github.com/tidyomics/tidyprint</a>
</li></ul>
</li>
<li>
<strong>proBatch</strong>: Analysis and correction of batch effects in high-throughput experiments.<ul><li>URL: <a href="https://github.com/Freddsle/proBatch" class="external-link uri">https://github.com/Freddsle/proBatch</a>
</li></ul>
</li>
<li>
<strong>StatescopeR</strong>: Discovering cell states from gene expression profiles and bulk RNA.<ul><li>URL: <a href="https://github.com/tgac-vumc/StatescopeR" class="external-link uri">https://github.com/tgac-vumc/StatescopeR</a>
</li></ul>
</li>
</ul>
</div>
<div class="section level2">
<h2 id="8-genetic-variation-and-splicing-impact">8. Genetic Variation and Splicing Impact<a class="anchor" aria-label="anchor" href="#8-genetic-variation-and-splicing-impact"></a>
</h2>
<ul>
<li>
<strong>fRagmentomics</strong>: Extraction of fragmentomic features and mutational status from cfDNA.<ul><li>URL: <a href="https://github.com/ElsaB-Lab/fRagmentomics" class="external-link uri">https://github.com/ElsaB-Lab/fRagmentomics</a>
</li></ul>
</li>
<li>
<strong>GXwasR</strong>: Conducting sex-aware genetic association analyses in complex traits.<ul><li>URL: <a href="https://github.com/boseb/GXwasR" class="external-link uri">https://github.com/boseb/GXwasR</a>
</li></ul>
</li>
<li>
<strong>SpliceImpactR</strong>: The impact of alternative RNA splicing on protein structure.<ul><li>URL: <a href="https://github.com/fiszbein-lab/SpliceImpactR" class="external-link uri">https://github.com/fiszbein-lab/SpliceImpactR</a>
</li></ul>
</li>
</ul>
</div>
<div class="section level2">
<h2 id="9-machine-learning-and-prediction-models">9. Machine Learning and Prediction Models<a class="anchor" aria-label="anchor" href="#9-machine-learning-and-prediction-models"></a>
</h2>
<ul>
<li>
<strong>asuri</strong>: Discovery of marker genes for survival and risk prediction.<ul><li>URL: <a href="https://github.com/jdelasrivas-lab/asuri" class="external-link uri">https://github.com/jdelasrivas-lab/asuri</a>
</li></ul>
</li>
<li>
<strong>dioscRi</strong>: Deep learning-based tool for clinical prediction in cytometry data.<ul><li>URL: <a href="https://github.com/ecool50/dioscRi" class="external-link uri">https://github.com/ecool50/dioscRi</a>
</li></ul>
</li>
<li>
<strong>CENTRE</strong>: Prediction of cell-type specific enhancer target interactions.<ul><li>URL: <a href="https://github.com/slrvv/CENTRE" class="external-link uri">https://github.com/slrvv/CENTRE</a>
</li></ul>
</li>
</ul>
</div>
<div class="section level2">
<h2 id="10-epigenomics-and-transcriptomics-analysis">10. Epigenomics and Transcriptomics Analysis<a class="anchor" aria-label="anchor" href="#10-epigenomics-and-transcriptomics-analysis"></a>
</h2>
<ul>
<li>
<strong>epiSeeker</strong>: Annotation and visualization of multi-omics epigenetic data.<ul><li>URL: <a href="https://github.com/YuLab-SMU/epiSeeker" class="external-link uri">https://github.com/YuLab-SMU/epiSeeker</a>
</li></ul>
</li>
<li>
<strong>decemedip</strong>: Deconvoluting cell types in MeDIP-seq data using a Bayesian model.<ul><li>URL: <a href="https://github.com/nshen7/decemedip" class="external-link uri">https://github.com/nshen7/decemedip</a>
</li></ul>
</li>
<li>
<strong>HumanRetinaLRSData</strong>: Dataset package for RNA sequencing and gene expression analysis.<ul><li>URL: <a href="https://github.com/sparthib/HumanRetinaLRSData" class="external-link uri">https://github.com/sparthib/HumanRetinaLRSData</a>
</li></ul>
</li>
</ul>
</div>
<div class="section level2">
<h2 id="11-variant-representation-and-interoperability">11. Variant Representation and Interoperability<a class="anchor" aria-label="anchor" href="#11-variant-representation-and-interoperability"></a>
</h2>
<ul>
<li>
<strong>AnVILVRS</strong>: Interface to the AnVIL VRS Toolkit for genomic variation representation.<ul><li>URL: <a href="https://github.com/Bioconductor/AnVILVRS" class="external-link uri">https://github.com/Bioconductor/AnVILVRS</a>
</li></ul>
</li>
<li>
<strong>immReferent</strong>: Interface for immune receptor (TCR/BCR) and HLA gene IMGT reference data.<ul><li>URL: <a href="https://github.com/BorchLab/immReferent" class="external-link uri">https://github.com/BorchLab/immReferent</a>
</li></ul>
</li>
</ul>
</div>
<div class="section level2">
<h2 id="12-cytometry-and-protein-interaction">12. Cytometry and Protein Interaction<a class="anchor" aria-label="anchor" href="#12-cytometry-and-protein-interaction"></a>
</h2>
<ul>
<li>
<strong>CompensAID</strong>: Detection of spillover errors in flow cytometry data.<ul><li>URL: <a href="https://github.com/Olsman/CompensAID" class="external-link uri">https://github.com/Olsman/CompensAID</a>
</li></ul>
</li>
<li>
<strong>SMTrackR</strong>: Protein-DNA binding and chromatin accessibility with DamID-seq data.<ul><li>URL: <a href="https://github.com/marshall-lab/damidBind" class="external-link uri">https://github.com/marshall-lab/damidBind</a>
</li></ul>
</li>
</ul>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <div class="links">
<h2 data-toc-skip>Links</h2>
<ul class="list-unstyled">
<li><a href="https://github.com/vjcitn/BiocRevSumm/" class="external-link">Browse source code</a></li>
<li><a href="https://github.com/vjcitn/BiocRevSumm/issues" class="external-link">Report a bug</a></li>
</ul>
</div>

<div class="license">
<h2 data-toc-skip>License</h2>
<ul class="list-unstyled">
<li><a href="https://www.r-project.org/Licenses/Artistic-2.0" class="external-link">Artistic-2.0</a></li>
</ul>
</div>


<div class="citation">
<h2 data-toc-skip>Citation</h2>
<ul class="list-unstyled">
<li><a href="authors.html#citation">Citing BiocRevSumm</a></li>
</ul>
</div>

<div class="developers">
<h2 data-toc-skip>Developers</h2>
<ul class="list-unstyled">
<li>Vince Carey <br><small class="roles"> Author, maintainer </small> <a href="https://orcid.org/0000-0003-4046-0063" target="orcid.widget" aria-label="ORCID" class="external-link"><span class="fab fa-orcid orcid" aria-hidden="true"></span></a>  </li>
</ul>
</div>



  </div>
</div>


      <footer><div class="copyright">
  <p></p>
<p>Developed by Vince Carey.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.2.0.</p>
</div>

      </footer>
</div>






  </body>
</html>
